Automattic/woocommerce-services
# Tweak/issue 1683

Fixes #1683 

This adds a tracking modal.

To test:
* Click on the "Track Packages" button on the top where "Create Shipping Label" used to be. This button will only show up when you have already created a label.
* Ensure the modal pops up with relevant tracking information.

TODO:
You will see the "Create Shipping Label" button before the "Track Packages" button. I will create a separate issue for this.
